### ISSUE TITLE
[Feature Request]: get only install data

### ISSUE THREAD

**Issue by nicoabie** [velut/node-query-registry#179]
First of all this is a great package.

I couldn't find a way "To request an abbreviated document with only the fields required to support installation"
https://github.com/npm/registry/blob/master/docs/responses/package-metadata.md

Is that possible?
if not can we add a flag or method?

**Comment by velut** [comment]
Thank you @nicoabie!

I've just added two new functions to retrieve abbreviated packuments: 
```typescript
getAbbreviatedPackument() => Promise<AbbreviatedPackument>
getRawAbbreviatedPackument() => Promise<RawAbbreviatedPackument>
```

Docs are available [here for getAbbreviatedPackument](https://www.jsdocs.io/package/query-registry/v/2.2.0-0#getAbbreviatedPackument) and [here for getRawAbbreviatedPackument](https://www.jsdocs.io/package/query-registry/v/2.2.0-0#getRawAbbreviatedPackument).

Example usage of `getAbbreviatedPackument`:

```typescript
import { getAbbreviatedPackument } from 'query-registry';

(async () => {
    const packument = await getAbbreviatedPackument({ name: 'query-registry' });

    // Output: 'query-registry'
    console.log(packument.name);
})();
```

You can try them by installing `query-registry@next` or directly [version `2.2.0-0`](https://www.npmjs.com/package/query-registry/v/2.2.0-0).

If everything looks good to you, let me know and I'll publish a stable release.

**Comment by nicoabie** [comment]
Looks great @velut!

Thanks!!!

**Comment by velut** [comment]
@nicoabie

I published the new stable version `2.2.0`.
